PrimeroAdministrationandConfigurationGuide.md

```diff
@@ -546,6 +569,10 @@ Once a Lookup is created, you can add to a field type and select whether you can
 
 To edit the properties of an existing Lookup, find the name and click _Edit_ in the options in the right column. From the _Show_ page, click the EDIT button at the top of the screen.
 
+## Jordan Case Identity
+
+* The ID in the cases list rows is either in the format ```case_ID_code/short_ID``` or just ```short_ID``` if the case_ID_code isn't set. This is set as an auto-populated field in the ```SystemSettings``` (this is for developers working on a configuration bundle)
```

Comment:
Needs a period at the end, but I would put all these developers-only notes at the beginning of their paragraphs.